.github/workflows/release.yml

```diff
@@ -212,10 +212,36 @@ jobs:
       run: |
         export TAG="$($HOME/.linkerd version --client --short)"
         go test -cover -race -v -mod=readonly ./cni-plugin/test -integration-tests
+  gh_release:
+    name: Create GH release
+    runs-on: ubuntu-18.04
+    needs: [kind_integration_tests, cloud_integration_tests]
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Pull CLI binaries
+      env:
+        TAG: ${{ github.ref }}
```

Comment:
Maybe keep using the `root-tag` script here like we do in the other jobs? We should never get to this step in the job and have `gitHub.ref` *not* reference the tag that triggered the workflow, but maybe just for sanity.